### ISSUE TITLE
Voiceover is not announcing the status messages "Tag saved" and "Tag deleted.": A11y_.NET 9 template_ManageMeta_ScreenReader

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#27463]
### GitHub Tags:
#A11yTCS; #A11yMAS; #WCAG4.1.3; #A11ySev3; #iOS; #SH_.NET9template_iOS_Jan2025; #Voiceover; #.NET Core; #Device; #screen reader; #StatusMessage; #AINotInscope; #A11yWCAG2.1; #iphone;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
iOS Version: 18.2
Screen reader: Voiceover

### Repro Steps:

1. Turn on Voiceover and Install and open ".NET 9 template" app.
2. "Home Screen" will open. Swipe till Hamburger menu and double tap on it, Swipe till manage meta and double tap on it.
3. Swipe till "Save" and "Delete" buttons present under 'Categories' and 'Tags' and double tap on it.
4. Observe that voiceover is not announcing the status messages "Tag saved" and "Tag deleted."

### Actual Result:
Voiceover is not announcing the status messages "Tag saved" and "Tag deleted."

**Note: Similar issue is observed in throughout the application for Saved, deleted status message.**

### Expected Result:
Voiceover should be announcing the status messages "Tag saved" and "Tag deleted."

### User Impact:
Screen reader users will not be able to understand whether the data is saved or not, and they will also not be able to understand if the data is deleted.

### Attachment:

https://github.com/user-attachments/assets/8d139fb0-9a47-46af-9bb2-2784445d10d0

**Comment by Yash14j** [comment]
#Rev:yaja;

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:03-05-25; #A11yFixed;

Verified the bug in environment
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment

https://github.com/user-attachments/assets/f5594d79-c026-43b9-8606-74646d3c1051
